[2021-08-04 13:29:38,053] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: my_dag.training_model_A 2021-08-04T13:29:32.937004+00:00 [queued]>
[2021-08-04 13:29:38,181] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: my_dag.training_model_A 2021-08-04T13:29:32.937004+00:00 [queued]>
[2021-08-04 13:29:38,181] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-08-04 13:29:38,182] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-08-04 13:29:38,182] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-08-04 13:29:38,267] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): training_model_A> on 2021-08-04T13:29:32.937004+00:00
[2021-08-04 13:29:38,318] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'training_model_A', '2021-08-04T13:29:32.937004+00:00', '--job-id', '1511', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpvdk0wcec', '--error-file', '/tmp/tmpnqccf62e']
[2021-08-04 13:29:38,319] {standard_task_runner.py:77} INFO - Job 1511: Subtask training_model_A
[2021-08-04 13:29:38,283] {standard_task_runner.py:52} INFO - Started process 10059 to run task
[2021-08-04 13:29:38,566] {logging_mixin.py:104} INFO - Running <TaskInstance: my_dag.training_model_A 2021-08-04T13:29:32.937004+00:00 [running]> on host 04479ea62028
[2021-08-04 13:29:38,918] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=training_model_A
AIRFLOW_CTX_EXECUTION_DATE=2021-08-04T13:29:32.937004+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-04T13:29:32.937004+00:00
[2021-08-04 13:29:38,919] {python.py:151} INFO - Done. Returned value was: 9
[2021-08-04 13:29:39,077] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=training_model_A, execution_date=20210804T132932, start_date=20210804T132938, end_date=20210804T132939
[2021-08-04 13:29:39,197] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-08-04 13:29:39,281] {local_task_job.py:149} INFO - Task exited with return code 0
